src/renderer/components/stake/Stake.tsx

```diff
@@ -19,10 +19,10 @@ export const Stake: React.FC<Props> = ({ ShareContent, TopContent, AddStake, has
 
   const tabs = useMemo(
     () => [
-      { label: intl.formatMessage({ id: 'common.add' }), key: 'add', content: <AddStake /> },
+      { label: intl.formatMessage({ id: 'common.add' }), key: 'add', content: <AddStake asset={asset} /> },
       { label: intl.formatMessage({ id: 'stake.withdraw' }), key: 'withdraw', content: 'withdraw' }
     ],
-    [intl]
+    [intl, asset]
   )
 
   const extra = useMemo(
```

Comment:
Is `{' '}` still needed in `line 40`?  _Can't set a comment there for any reason ..._